### PR TITLE
Fix some addon.xml issues

### PR DESCRIPTION
**Pull request by jlaur**
Regression of #13289, #14235

### CODE REVIEW

**Review by lolodomo — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml:1]: This should not be named `binding.xml`.

Comment (jlaur) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml:1]: I agree it's a bit strange/awkward having **binding** as thing-type, but is the filename actually a problem? What would you suggest?

Comment (J-N-K) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml:1]: The most common name seems to be `thing-types.xml`.

Comment (lolodomo) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml:1]: thing-types.xml is the default file name (bonding skeleton) and what is commonly used by a lot of bondings.

Comment (jlaur) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml:1]: We could prefix the filenames with "thing-type-"? This binding has a separate file per thing type.

Comment (lolodomo) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml:1]: Ok sorry, I understood. The thing type id is "binding"! What a strange idea! It should have been noticed at review time. 
Maybe use as filename what should have been used as thing type id?

Comment (lolodomo) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml:1]: Looking quickly at the code, my feeling is that this thing type should have never existed. It was created apparently just to provide information about the binding to the user. 

You could call the file binding-information.xml for example.

Comment (andrewfg) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml:1]: Yeah. I think Guether created it in order to help him debug issues raised on the community forum. Specifically it puts the OH build version into the log file so one can easily see the context just from the log itself.

Comment (jlaur) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/binding.xml]: > You could call the file binding-information.xml for example.

That would be inconsistent as the other files are named 1:1 as their thing type id. I have prefixed all filenames with "thing-type-", let me know what you think.

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**